picard/tagger.py

```diff
@@ -1062,7 +1062,7 @@ def _lookup_disc(self, disc, result=None, error=None):
         self.restore_cursor()
         if error is not None:
             QtWidgets.QMessageBox.critical(self.window, _("CD Lookup Error"),
-                                           _("Error while reading CD:\n\n%s") % error)
+                                           _("Error while reading CD:\n\n%s"), error)
```

Comment:
I think this change is incorrect, our log.* functions support passing format arguments but here that's a Qt message box.

Reply:
Right, my overlook